File: 190. Reverse Bits.cpp
class Solution {  
public:  
    uint32_t reverseBits(uint32_t n) {  
        uint32_t res = 0;  
        for (uint32_t i = 0; i < 32; i++)
            res = (res << 1) | ((n >> i) & 1);
        return res;  
    }  
}; 

File: 190. Reverse Bits.py
class Solution(object):
    def reverseBits(self, n):
        """
        :type n: int
        :rtype: int
        """
        bin_n = bin(n)[2:][::-1]
        while len(bin_n) < 32:
            bin_n += "0"
        return int(bin_n,2)
